packages/kolibri/uiText/commonCoreStrings.js

```diff
@@ -565,6 +565,10 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
     message: 'Important: please remember this account information. Write it down if needed.',
     context: 'Helper/information text to remind user to take note of their account information.',
   },
+  quizNotStartedText: {
+    message: 'Quiz not started',
```

Comment:
Just wondering if this can sit commonCoach Or searchAndFilter?

Reply:
It could perhaps go in searchAndFilter. My first thought was to put this string in commonCoach, but since it’s primary usage would be the QuizReport in kolibri-common, I figured it would be best to place it in a more general place. I guess it’s not super clear to me where this string should go since it’s not within the release feature. Should I move this string? @marcellamaki